src/cdk/v2/destinations/intercom/procWorkflow.yaml

```diff
@@ -208,9 +217,25 @@ steps:
       response.method = "POST";
       response.userId = .message.anonymousId;
       $.context.response.push(response);
+      payload = response.body.JSON;
+      const companyId = await $.createOrUpdateCompany(payload, .destination);
+      $.assert(companyId, "Unable to create or update company");
```

Comment:
Why again we are doing this call here?

Reply:
We are doing this call only one time from transformer as we are sending response with statusCode 299. This `groupPayloadForOlderVersion` will only be executed for group call when we select apiVersion 1.4 in UI.